data_diff/databases/bigquery.py

```diff
@@ -76,6 +76,11 @@ class Dialect(BaseDialect):
     }
     TYPE_ARRAY_RE = re.compile(r"ARRAY<(.+)>")
     TYPE_STRUCT_RE = re.compile(r"STRUCT<(.+)>")
+    # [BIG]NUMERIC, [BIG]NUMERIC(precision, scale), [BIG]NUMERIC(precision)
+    TYPE_NUMERIC_RE = re.compile(r"^((BIG)?NUMERIC)(?:\((\d+)(?:, (\d+))?\))?$")
```

Comment:
Is it guaranteed to have strictly 1 space after the comma? Never 0?

Reply:
Yes, the space is always there even if it user didn't add it explicitly.